101-clickable-grid/clickable-grid.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <style>
+        .content>div{
+            border:1px solid #ff3300;
+            width:1200px;
+            display:flex;
+            justify-content: left;
+            flex-wrap: wrap;
+        }
+        .content>div>div{
+            display: flex;
+            box-sizing:border-box;
+            margin:5px;
+            height:40px;
+            background:darkgoldenrod;
+            width:40px;
+
+
+        }
+    </style>
+</head>
+<body>
+<button>++</button>
+<span>1</span>
+<button>---</button>
+<div class="content">
+</div>
+<script>
+window.onload = function () {
+    var btns = document.querySelectorAll('button'),
+        addBtn = btns[0],
+        subBtn = btns[1],
+        numSpan = document.querySelector('span'),
+        contentDiv = document.querySelector('.content');
+
+    var currentNum = 1;
+    addBtn.onclick = function () {
+        currentNum++;
+        numSpan.innerText = Math.pow(currentNum,2);
+        updateDiv(numSpan.innerText)
+    }
+    subBtn.onclick = function () {
+        currentNum--;
+        numSpan.innerText = Math.pow(currentNum,2);
+        updateDiv(numSpan.innerText)
+
+    }
+
+
+    function updateDiv(num){
```

Comment:
given you are passing in `numSpan.innerText`, maybe you should check whether num is a number or not here